src/psyclone/dynamo0p3.py

```diff
@@ -2167,7 +2167,7 @@ class DynReferenceElement(DynCollection):
     # pylint: disable=too-many-instance-attributes
```

Comment:
Not necessarily yours but pylint complains:

    dynamo0p3.py:81:0: W0611: Unused INTEGER_SINGLE_TYPE imported from psyclone.psyir.symbols (unused-import)

Reply:
Ah ... I think I fixed that in one of my later branches, must have missed it here - thanks!